test/test_other.py

```diff
@@ -14008,3 +14009,9 @@ def test_noExitRuntime_deps(self):
 ''')
     err = self.expect_fail([EMCC, test_file('hello_world.c'), '--js-library=lib.js', '-sDEFAULT_LIBRARY_FUNCS_TO_INCLUDE=foo'])
     self.assertContained('error: noExitRuntime cannot be referenced via __deps mechansim', err)
+
+  def test_hello_world_argv(self):
+    self.do_runf(test_file('hello_world_argv.c'), 'hello, world! (1)')
```

Comment:
This one passes ok, but the one below will fail.  Specifically you have to do `-sENVIRONMENT=web -sSTRICT --closure=1`.  

In particular note the `-sENVIRONMENT=web` .. if you include node support the issue with `arguments_` doesn't show up because node support assigs to this variable.

The reason I added two version of this test is that I've often wanted a simple test that just does hello world but includes argv.